k8s/base/worker.yaml

```diff
@@ -45,7 +45,7 @@ spec:
             "--evm-chain-id", "$(EVM_CHAIN_ID)",
             "--bind-address", "$(BIND_ADDRESS)",
             "--bind-health-address", "$(BIND_HEALTH_ADDRESS)",
-            "--w3s-token", "$(W3S_TOKEN)",
+            "--basin-w3s-endpoint", "$(BASIN_W3S_ENDPOINT)",
```

Comment:
new flag to indicate where the server is located. right now it's a private server inside kubernetes reachable at `basin-w3s.basin-staging.svc.cluster.local:8080`